vitrina/orgs/migrations/0006_auto_20220919_1116.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.15 on 2022-09-19 08:16
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import migrations
+
+
+def update_user_field(apps, schema_editor):
+    User = apps.get_model('vitrina_users', 'User')
+    Representative = apps.get_model('vitrina_orgs', 'Representative')
+
+    for rep in Representative.objects.all():
+        if rep.email:
+            try:
+                user = User.objects.get(email=rep.email)
+                rep.user = user
+                rep.role = user.role
+                rep.save()
+            except ObjectDoesNotExist:
+                pass
```

Comment:
It should be opposite, Representatives should be associated with users, by email. One user can be member of multiple organizations. `User` itself does not have any relation to organization. User relation to organization is defined through `Representative`.